test/generateTasks.spec.js

```diff
@@ -21,7 +21,7 @@ const files = [
   '.hidden/test.txt'
 ]
 
-const workDir = path.join(process.env.HOME, 'tmp-lint-staged')
+const workDir = path.join(require('os').homedir(), 'tmp-lint-staged')
```

Comment:
Let's follow consistent `import` style. Please import the `os` package at the top. Also, I think `os.tmpdir()` would be more appropriate for our use-case.

Reply:
@sudo-suhas why do you think `tmpdir` is more appropriate?

Reply:
Ah never mind my comment. I saw it's in the test

Reply:
:relaxed: it's for an imaginary directory..